<ui version="4.0" >
 <class>startNetworkGameDialog</class>
 <widget class="QDialog" name="startNetworkGameDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Start Network Game</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Maximal Player Number:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>181</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_startGame" >
       <property name="text" >
        <string>Start Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QTreeWidget" name="treeWidget" >
     <column>
      <property name="text" >
       <string>    Connected Player    </string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="label_maxPlayerNumber" >
     <property name="text" >
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>pushButton_startGame</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_startGame</sender>
   <signal>clicked()</signal>
   <receiver>startNetworkGameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>140</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel" >
     <x>75</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>startNetworkGameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>235</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel" >
     <x>276</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
